lib/routes/ups/track.ts

```diff
@@ -0,0 +1,107 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
+    path: '/track/:trackingNumber',
+    categories: ['other'],
+    example: '/ups/track/1Z78R6790470567520',
+    parameters: { trackingNumber: 'The UPS tracking number (e.g., 1Z78R6790470567520).' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['ups.com/track?loc=en_US&tracknum=:trackingNumber'],
+            target: '/ups/track/:trackingNumber',
+        },
+    ],
+    name: 'UPS Tracking',
```

Comment:
Do not repeat namespace name https://github.com/DIYgod/RSSHub/blob/612d9cd34b99ceb42759180c6e91657af34fa843/lib/routes/ups/namespace.ts#L4 in route name